contracts/package.json

```diff
@@ -22,6 +22,7 @@
     "abi/"
   ],
   "devDependencies": {
+    "@arbitrum/nitro-contracts": "^1.0.0-beta.7",
```

Comment:
hmm, not sure if we should depend on beta version.

Reply:
Done